agent-instrumentation/src/jvmMain/kotlin/com/epam/drill/agent/instrument/servers/JettyTransformerObject.kt

```diff
@@ -38,11 +39,17 @@ abstract class JettyTransformerObject(
         val agentIdHeader = headersRetriever.agentIdHeader()
         val agentIdValue = headersRetriever.agentIdHeaderValue()
         logger.info { "transform: Starting JettyTransformer with admin host $adminUrl..." }
-        val method =
+        val method = try {
             ctClass.getMethod(
                 "handle",
                 "(Ljava/lang/String;Lorg/eclipse/jetty/server/Request;Ljavax/servlet/http/HttpServletRequest;Ljavax/servlet/http/HttpServletResponse;)V"
             )
+        } catch (e: NotFoundException) {
+            ctClass.getMethod(
```

Comment:
Add comment explaining behavior.
Something along the lines of `// Instrumentation for Java X From Java X HttpServletRequest and HttpServletResponse where relocated from javax to jakarta package` (place actual java version instead of X)

Reply:
done